docs/mkdocs.yml

```diff
@@ -137,6 +152,22 @@ markdown_extensions:
 plugins:
   - table-reader
   - search
+  - git-revision-date-localized:
+      enable_creation_date: true
+  - git-committers:
```

Comment:
I don't think this is a good idea, it is showing inactive contributors and Pwuts.

Reply:
done